rapids-cmake/export/find_package_root.cmake

```diff
@@ -29,7 +29,8 @@ needs to be set to the provided path.
   rapids_export_find_package_root( (BUILD|INSTALL)
                                    <PackageName>
                                    <directory_path>
-                                   <ExportSet>
+                                   (<ExportSetName> | EXPORT_SET [ExportSetName])
```

Comment:
Is this supporting both uses for backwards compatibility? If so, can we make a plan to deprecate the former usage? Maybe using `rapids_cmake_policy`?

Reply:
Yes it is for backwards compatibility, as RAFT, cudf, cuml, etc use this function. 

Deprecation does sound reasonable. Depreacte in 23.12 and remove in the second release of 24.X feels reasonable to me.

Reply:
Looking at keeping symetry across the `rapids_export` functions I am also going to update `rapids_export_find_package_root`